test/reducers/columnStateHelper-test.js

```diff
@@ -7,192 +7,6 @@ import emptyFunction from 'emptyFunction';
 import { assert } from 'chai';
 
 describe('columnStateHelper', function() {
-  describe('resizeColumn', function() {
```

Comment:
what was the reason for removing these?

Reply:
Now, That method doesn't exist in columnStateHelper